### PR TITLE
Php81 + Weak ReCaptcha Dependency

### PR DESCRIPTION
**Pull request by amenk**

### CODE REVIEW

**Comment by amenk** [comment]
@DanieliMi please review my last commit

File: Observer/RecaptchaFormObserver.php [hunk 1]
```diff
@@ -2,47 +2,53 @@
...
-            $request = $observer->getRequest();
-            $this->requestHandler->execute(
-                self::RECAPTCHA_FORM_KEY,
-                $request,
-                $this->httpResponse,
-                $this->redirect->getRefererUrl()
-            );
+        if (!($this->isCaptchaEnabled !== null
+            && $this->requestHandler !== null
+            && $this->isCaptchaEnabled->isCaptchaEnabledFor(self::RECAPTCHA_FORM_KEY))) {
```
Comment (DanieliMi) [inline Observer/RecaptchaFormObserver.php:42]: Just formatting and with an `or` it seems a bit easier to read, but that is just nitpicking.
```
        if (
            $this->isCaptchaEnabled === null
            || $this->requestHandler === null
            || !$this->isCaptchaEnabled->isCaptchaEnabledFor(self::RECAPTCHA_FORM_KEY)
            ) {
```

Comment (amenk) [reply Observer/RecaptchaFormObserver.php:42]: true, hm ..

Comment (amenk) [reply Observer/RecaptchaFormObserver.php:42]: maybe another time :)

**Pull request merged**